cabal.project

```diff
@@ -1,14 +1,19 @@
 packages: .
 
+package aeson
+    flags: +cffi
+
+debug-info: True
+
 source-repository-package
     type: git
     location: https://github.com/kadena-io/chainweb-node.git
-    tag: c9d8c1112ef0093bca04f6124721d8fa41ebf5a9
+    tag: 3c5f2264520a3edccf4731436b3a0545098d85a9
 
 source-repository-package
     type: git
     location: https://github.com/kadena-io/pact.git
-    tag: ace18a2489bf86595af09316bd1c09794c326b77
+    tag: 2ca4cae3eb8a0ef2783aa9457388c31b44a0dfbc
```

Comment:
Same here for whatever version of pact chainweb 2.0 is using

Reply:
That one is already up to date